docs/rest-api/rest-notebook.md

```diff
@@ -106,6 +106,288 @@ limitations under the License.
   <table class="table-configuration">
     <col width="200">
     <tr>
+      <th>Get notebook</th>
+      <th></th>
+    </tr>
+    <tr>
+      <td>Description</td>
+      <td>This ```GET``` method retrieves an existing notebook's information using the given id.
+          The body field of the returned JSON contain information about paragraphs in the notebook.
+      </td>
+    </tr>
+    <tr>
+      <td>URL</td>
+      <td>```http://[zeppelin-server]:[zeppelin-port]/api/notebook/[notebookId]```</td>
+    </tr>
+    <tr>
+      <td>Success code</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <td> Fail code</td>
+      <td> 500 </td>
+    </tr>
+    <tr>
+      <td> sample JSON response </td>
+      <td><pre>
+{
+  "status": "OK",
+  "message": "",
+  "body": {
+    "paragraphs": [
```

Comment:
I just keep information of the first paragraph and remove others. Please let me know if it is still too long.